airbyte-integrations/connectors/source-facebook-marketing-api-singer/build.gradle

```diff
@@ -0,0 +1,34 @@
+project.ext.pyModule = 'source_facebook_marketing_api_singer'
+apply from: rootProject.file('tools/gradle/commons/integrations/python.gradle')
+apply from: rootProject.file('tools/gradle/commons/integrations/image.gradle')
+apply from: rootProject.file('tools/gradle/commons/integrations/test-image.gradle')
+apply from: rootProject.file('tools/gradle/commons/integrations/integration-test.gradle')
+apply from: rootProject.file('tools/gradle/commons/integrations/standard-source-test-python.gradle')
+
+
+standardSourceTestPython {
+    ext {
+        imageName = "${extractImageName(project.file('Dockerfile'))}:dev"
+        pythonContainerName = "${extractImageName(project.file('Dockerfile.test'))}:dev"
+    }
+}
+
+task installTestDeps(type: PythonTask){
+    module = "pip"
+    command = "install .[tests]"
+}
+
+task unitTest(type: PythonTask){
```

Comment:
when do we use this versus `command = "setup.py test"`? it seems like we do both things.

Reply:
also curious about this

Reply:
we only give guidance to use `pytest` (or `python -m pytest`, they're [99% equivalent](https://docs.pytest.org/en/stable/pythonpath.html#invoking-pytest-versus-python-m-pytest)) now because `setup.py test` [is deprecated.](https://setuptools.readthedocs.io/en/latest/userguide/commands.html#test-build-package-and-run-a-unittest-suite)